src/typeDefs/schema.graphql

```diff
@@ -159,6 +160,41 @@ input SubmissionSearchFilters {
   offset: Int
 }
 
+type AuditLogs {
+  id: ID!
+  actionType: ActionType!
+  tableName: TableName!
```

Comment:
I  realized this is a mistake on my requirements I gave here. We shouldn't expose our tables to the public. (security vulnerability)
Let's change this to `entity` or `model`, `objectType` or something like that. the enum values can still be reused